customTests/azure_common.nhc

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
 export ALL_REDUCE_PATH=/opt/nccl-tests/build/all_reduce_perf
-export HPCX_DIR=/opt/hpcx
+
+function get_sku(){
```

Comment:
duplicate function here?

Reply:
ah I'm not sure how this happened let me address.